README.md

```diff
@@ -94,6 +94,12 @@ making the combined instantiation/stub declaration very terse:
 double(:foo => 'bar')
 ```
 
+So, the above examples can be written simply as:
+
+```ruby
+allow(book).to receive(:title).and_return(book)
+```
+
```

Comment:
Sure. So I'm a relatively new programmer with a writing and logic background. I think this is clearer because it actually connects the dots for people reading this documentation and it's the more fleshed out version of the conclusion that you're alluding to. It answers the question "Can you simply pass in book even though it seems to have an extra argument which functions as a name? Is this what they mean?" I think the extra information is harmless, non-verbose, and finishes the job.